pandas/core/computation/pytables.py

```diff
@@ -226,7 +226,11 @@ def stringify(value):
             if v not in metadata:
                 result = -1
             else:
-                result = metadata.searchsorted(v, side="left")
+                # error: Incompatible types in assignment (expression has type
+                # "Union[Any, ndarray[Any, Any]]", variable has type "int")
+                result = metadata.searchsorted(
+                    v, side="left"
+                )  # type: ignore[assignment]
```

Comment:
can avoid the ignore using a variable annotation for `result`?

Reply:
Done in next commit